src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -400,25 +403,23 @@ public StudyEntity doGetStudyWithPreFetchedLoadFlowResultAndUpdateIsPrivate(UUID
                 .log(ROOT_CATEGORY_REACTOR, Level.FINE);
     }
 
-    private Mono<? extends Throwable> handleStudyCreationError(String studyName, ClientResponse clientResponse) {
+    private Mono<? extends Throwable> handleStudyCreationError(String studyName, String userId, boolean isPrivate, ClientResponse clientResponse) {
         return clientResponse.bodyToMono(String.class).flatMap(body -> {
             try {
-                String message;
                 JsonNode node = new ObjectMapper().readTree(body).path("message");
                 if (!node.isMissingNode()) {
-                    message = node.asText();
-                    emitStudyError(studyName, UPDATE_TYPE_STUDIES, message);
+                    emitStudyCreationError(studyName, userId, isPrivate, node.asText());
                 }
             } catch (JsonProcessingException e) {
                 if (!body.isEmpty()) {
-                    emitStudyError(studyName, UPDATE_TYPE_STUDIES, body);
+                    emitStudyCreationError(studyName, userId, isPrivate, body);
```

Comment:
maybe in another PR and in another repo, but I think we should not send error notifications to other users for public studies.

Reply:
Done